src/integrationTest/resources/application-integration-test.yml

```diff
@@ -88,3 +88,8 @@ service.base-url: http://localhost:8080
 
 pes:
   contract-start-date: 2025-10-01
+
+induction-extension:
+  periods:
+    - start: 2024-12-20
+      end: 2025-01-01
```

Comment:
Config is for last year for tests so the build doesn't fail when we do builds on the extension period